src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -50,12 +50,13 @@ public Intake(int intakeMotorID, MotorType motorType, int forwardChannel, int re
    * Sets the intake motor speed.
    * 
    * @param intakeSpeed The speed of the intake motor
```

Comment:
Changes to input variables should be reflected in the javadocs
```suggestion
   * @param newSpeed The speed of the intake motor
```